frontstage/create_app.py

```diff
@@ -46,6 +50,8 @@ def create_app_object():
     @app.after_request
     def apply_caching(response):
```

Comment:
As `X-Frame-Options` isn't a cache header the method should be renamed something like `apply_headers`